composer.json

```
@@ -10,13 +10,13 @@
     }
   ],
   "require": {
-    "alleyinteractive/mantle-framework": "dev-main",
+    "alleyinteractive/mantle-framework": "dev-unit-test-standalone",
```

change back after https://github.com/alleyinteractive/mantle-framework/pull/120 is merged

```suggestion
    "alleyinteractive/mantle-framework": "dev-main",
```